gradle.properties

```diff
@@ -1,3 +1,4 @@
+description=Kestra's NATS Plugin enables high-performance communication in distributed systems for enhanced workflow management and seamless messaging.
```

Comment:
Did you check with Martin ?
As we should add the word Edge somewhere as it's the buzzword behind NATS ;)

Reply:
That's Martin's text there, I have to admit I did a dumb raw copy paste, maybe we can discuss about it with him ?

Reply:
If it's Martin's text, let it be